lutris/util/wine/proton.py

```diff
@@ -52,35 +52,12 @@ def get_umu_path() -> str:
     The path that this returns will be considered an Umu path by is_umu_path().
 
     If this script can't be found this will raise MissingExecutableError."""
-
-    # We can access Umu via it's main .py file, or via the public name which is just
-    # symbolic link to the same file. We'll check both for compatibility, just in case
-    # Umu gets translated into Perl or something.
-    entry_points = ["umu_run.py", "umu-run"]
-
-    custom_path = settings.read_setting("umu_path")
```

Comment:
You've removed support for the umu_path setting altogether. Why?

Reply:
I'll revert this change as this was careless of me and will look into how it works

Reply:
I believe this is a setting you'd have to edit your configuration files to set, but I'd expect it to grow a GUI setting in due course.